recipes/LibriMix/separation/hparams/sepformer-libri3mix.yaml

```diff
@@ -111,7 +111,7 @@ SBtfinter: !new:speechbrain.lobes.models.dual_path.SBTransformerBlock
     d_model: !ref <out_channels>
     nhead: 8
     d_ffn: 1024
-    dropout: 0
+    dropout: 0.0
```

Comment:
Does it break if you keep an integer here?

Reply:
Yep